### PR TITLE
connecting to our backend for data, some styling, etc

### PR DESCRIPTION
**Pull request by imriven**
- Calling the WEB API for Categorized Spending Data
- Styling to plotly graph: spacing, responsiveness, etc.

https://trello.com/c/VhehpBNy/109-connecting-ds-bar-chart-to-our-front-end-dash-board

### CODE REVIEW

**Comment by imriven** [comment]
[Connecting DS categories data to web be, to the web fe  Dash board and component](https://trello.com/c/VhehpBNy/109-connecting-ds-bar-chart-to-our-front-end-dash-board)

**Review by chelsabeth — APPROVED** [review]
Code looks clean and good 👍 Great work

File: src/components/pages/CategorizedSpending/CategorizedSpending.js [hunk 1]
```diff
@@ -42,117 +40,22 @@ const mapStateToProps = state => {
...
-              <Link to="/" className="button-to-home">
-                Return Home
-              </Link>
-            </StyledDiv>
-          ) : (
-            <Loading />
-          )}
+          <StyledDiv>
+            <h1>Spending By Category</h1>
+            {/* <h1>Your Spending For {chosenMonth}</h1> */}
```
Comment (Cireimu) [inline src/components/pages/CategorizedSpending/CategorizedSpending.js:52]: Might want to remove this commented out piece of code.

**Review by Cireimu — APPROVED** [review]
Looks good, the only thing that should be changed if anything is that you have some commented out code.

**Pull request merged**